css/globals.less

```diff
@@ -10,6 +10,7 @@
 @hylo-teal: #4fb19f;
 @hylo-orange: #e79336;
 @hylo-green: #52b065;
+@button-border: #B3B3B3;
```

Comment:
where's this from? bootstrap, or our own styles from hylo-frontend? if the latter, let's just use `@light-gray-border` instead, which matches bootstrap